src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1201,31 +1222,18 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             });
         }
 
-        [ActiveIssue(22163, TargetFrameworkMonikers.Uap)]
-        [OuterLoop] // TODO: Issue #11345
-        [Fact]
-        public async Task GetAsync_StatusCode99_ExpectedException()
-        {
-            await LoopbackServer.CreateServerAsync(async (server, url) =>
-            {
-                using (var client = new HttpClient())
-                {
-                    Task<HttpResponseMessage> getResponse = client.GetAsync(url);
-                    await LoopbackServer.ReadRequestAndSendResponseAsync(server,
-                            $"HTTP/1.1 99\r\n" +
-                            $"Date: {DateTimeOffset.UtcNow:R}\r\n" +
-                            "\r\n");
-
-                    await Assert.ThrowsAsync<HttpRequestException>(() => getResponse);
-                }
-            });
-        }
-
         [OuterLoop] // TODO: Issue #11345
         [Theory]
+        [InlineData(99)]
         [InlineData(1000)]
         public async Task GetAsync_StatusCodeOutOfRange_ExpectedException(int statusCode)
         {
+            if (PlatformDetection.IsUap && statusCode == 99)
+            {
+                // UAP platform allows this status code due to historical reasons.
```

Comment:
Can we fix this in CoreFX code? (still throw even if the underlying HTTP stack didn't error?)